spec/services/types_generator_spec.rb

```diff
@@ -523,7 +523,7 @@
               {
                 structuredValue: [
                   {
-                    value: 'Software/Code',
+                    value: 'Software/<wbr>Code',
```

Comment:
This is incorrect. There should be no HTML here.

Reply:
We do the lookup based on the label, so without this, the look fails. I'm wondering, based on that, if we should just remove the `/Code` because it's the problem here and would then require code to strip the `<wbr>` before the lookup... seems wrong to me

Reply:
Have you considered adding a new separate `html_label` property to the WorkType model?